app/__init__.py

```
@@ -3,12 +3,13 @@
 App module initialization
 """
 
+from importlib.machinery import ModuleSpec
 import importlib.util
 import os
 import sys
-from importlib.machinery import ModuleSpec
 from typing import Optional, cast
```

Flake8 (7.3.0)

[error] 'typing.Optional' imported but unused

(F401)